packages.yaml

```diff
@@ -123,6 +123,24 @@ packages:
       - py-virtualenv@15.1.0
       - py-pytest@3.0.7
 
+  # List of python 2 only packages that do not require
+  # BLAS / LAPACK
+  serial_python_2:
+    target_matrix:
+      - gnu-stable
+      - gnu-bleeding-edge
+      - gnu-gpu
+      - intel
+    target_filter:
+      python: ['python@2.7.13']
+    requires:
+      - architecture
+      - compiler
+      - python
+    specs:
+      - py-macs@1.4.2-1
+      - py-macs2@2.1.1.20160309
```

Comment:
This one is wrong. `py-macs2` requires `lapack` through numpy`.

Reply:
Ok, I just checked. pymacs2 was already installed. Is there something new about this package?